detector/detector.go

```diff
@@ -399,9 +419,32 @@ func metricsHandler(registry *prometheus.Registry) http.Handler {
 }
 
 func registerMetrics() *prometheus.Registry {
+	counterOpts := prometheus.CounterOpts{}
+	gaugeOpts := prometheus.GaugeOpts{}
+
+	if nomadDc, ok := os.LookupEnv("NOMAD_DC"); ok {
```

Comment:
Shall we do a check to cover the empty case?

```
if nomadDC, ok := os.LookupEnv("NOMAD_DC"); ok && nomadDC != "" {
   ...
}
```

While golang prefers camelCase for variable names, for acronyms like URL or HTTP, it's all caps e.g. ServeHTTP.
Shall we rename it to `nomadDC`? (Just a suggestion, feel free to ignore 🙂).

[`Reference blog post`](https://blog.devgenius.io/write-better-go-code-by-using-consistent-case-for-initials-in-variables-b84e862fd0ba)

Reply:
Thanks, I think this is more consistent, I've renamed the variable.